tests/rules/sort-keys.test.js

```diff
@@ -0,0 +1,2035 @@
+		},
+		{
+			code: '{"b_":1, "b":3, "a":2}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+		{
+			code: '{"c":2, "C":3, "b_":1}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+		{
+			code: '{"C":2, "c":3, "b_":1}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+		{
+			code: '{"a":4, "A":3, "_":2, "$":1}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+		{
+			code: '{"A":3, "11":2, "2":4, "1":1}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+		{
+			code: '{"è":4, "À":3, "Z":2, "#":1}',
+			options: ["desc", { natural: true, caseSensitive: false }],
+		},
+
+		// desc, natural, insensitive, minKeys should ignore unsorted keys when number of keys is less than minKeys
+		{
+			code: '{"a":1, "_":2, "b":3}',
+			options: [
+				"desc",
+				{ natural: true, caseSensitive: false, minKeys: 4 },
+			],
+		},
+
+		// allowLineSeparatedGroups option
+		{
+			code: `
+						{
+								"a": 1,
+								"b": 2,
+								"c": 3,
+								"e": 4,
+								"f": 5,
+								"g": 6
+						}
+				`,
+			options: ["asc", { allowLineSeparatedGroups: false }],
+		},
+		{
+			code: `
+						{
+								"e": 1,
+								"f": 2,
+								"g": 3,
+
+								"a": 4,
+								"b": 5,
+								"c": 6
+						}
+				`,
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+		{
+			code: `
+						{
+								"b": 1,
+
+								// comment
+								"a": 2,
+								"c": 3
+						}
+				`,
+			language: "json/jsonc",
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+		{
+			code: `
+						{
+								"b": 1
+
+								,
+
+								// comment
+								"a": 2,
+								"c": 3
+						}
+				`,
+			language: "json/jsonc",
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+		{
+			code: `
+						{
+								"b": "/*",
+
+								"a": "*/"
+						}
+				`,
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+		{
+			code: `
+						{
+							"b":1,
+							"c": {
+								"y":1,
+								"z":1,
+
+								"x":1
+							},
+
+							"a":1
+						}
+				`,
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+
+		{
+			code: `
+						{
+							"b":1,
+							a: {
+								"y":1,
+								x:1,
+
+								"z":1
+							},
+
+							c:1
+						}
+				`,
+			language: `json/json5`,
+			options: ["desc", { allowLineSeparatedGroups: true }],
+		},
+
+		// Commas are not considered separating lines
+		{
+			code: `
+			{
+				"b": 1
+
+				,
+
+				"a": 2
+			}
+			`,
+			options: ["asc", { allowLineSeparatedGroups: true }],
+		},
+		{
+			code: `
+			{
+				"a": 1
+
+
+				,
+				"b": 2
+			}
+			`,
+			options: ["asc", { allowLineSeparatedGroups: false }],
+		},
+		{
+			code: `
+						{
+								"b": 1
+						// comment before comma
+
+						,
+						"a": 2
+						}
+				`,
+			language: "json/jsonc",
+			options: ["asc", { allowLineSeparatedGroups: true }],
+			languageOptions: { ecmaVersion: 6 },
```

Comment:
```suggestion
```

Reply:
Updated https://github.com/eslint/json/pull/76/commits/765cbde30b90c868cac9315a59072800a0c697be That was left over from copying from eslint/sort-keys